### PR TITLE
ActivityItem: Adding theme to enable theme switching.

### PR DESCRIPTION
**Pull request by marygans**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ yarn change`

#### Description of changes

Added theme to API to enable theme switching for ActivityItem. Previously palette colors were not inverting on dark mode. 

**Before:**
Screen Shot 2020-03-02 at 7 20 51 PM

**After:**
Screen Shot 2020-03-02 at 7 22 03 PM

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/12157)

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/ActivityItem/ActivityItem.tsx [hunk 1]
```diff
@@ -3,12 +3,14 @@ import { IActivityItemProps } from './ActivityItem.types';
...
 import { getStyles } from './ActivityItem.styles';
 import { PersonaSize, PersonaCoin, IPersonaSharedProps, IPersonaCoinProps } from '../../Persona';
+import { customizable } from '../../Utilities';
 
 type OptionalReactKey = { key?: React.Key };
 
 /**
  * {@docCategory ActivityItem}
  */
+@customizable('ActivityItem', ['theme', 'styles'], true)
```
Comment (aneeshack4) [inline packages/office-ui-fabric-react/src/components/ActivityItem/ActivityItem.tsx]: this is why the bundle size increased... it should be fine but perhaps we can consider alternatives?

Comment (marygans) [reply packages/office-ui-fabric-react/src/components/ActivityItem/ActivityItem.tsx]: Oh I see, hmm do you have any ideas on alternatives for using this decorator?

Comment (marygans) [reply packages/office-ui-fabric-react/src/components/ActivityItem/ActivityItem.tsx]: Hey @aneeshack4! do you have any suggestions on alternatives?

**Comment by ecraig12345** [comment]
@marygans Do you plan to finish this, or should we close it? (Or close and then reopen later when you're ready to work on it.)

**Pull request closed without merging**